src/platform/datapath_winuser.c

```diff
@@ -76,13 +76,13 @@ CxPlatFuzzerRecvMsg(
 //
 #define URO_MAX_DATAGRAMS_PER_INDICATION    64
 
-static_assert(
+CXPLAT_STATIC_ASSERT(
     sizeof(QUIC_BUFFER) == sizeof(WSABUF),
     "WSABUF is assumed to be interchangeable for QUIC_BUFFER");
-static_assert(
+CXPLAT_STATIC_ASSERT(
```

Comment:
That didn't help. `clang-cl` still says:
```c
platform/datapath_winuser.c(84,5): error: static_assert expression is not an integral constant expression
    FIELD_OFFSET(QUIC_BUFFER, Length) == FIELD_OFFSET(WSABUF, len),
    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
f:\ProgramFiler-x86\WindowsKits\Include\10.0.19041.0\um\winnt.h(1205,38): note: expanded from macro 'FIELD_OFFSET'
#define FIELD_OFFSET(type, field)    ((LONG)(LONG_PTR)&(((type *)0)->field))                                     ^
```

Although the preprocessed output of line 84 seems correct:
```
_Static_assert ( ( (LONG) (LONG_PTR) & ( ( (QUIC_BUFFER *) 0)->Length)) == 
   ( (LONG) (LONG_PTR) & ( ( (WSABUF *) 0)->len)),  "WSABUF is assumed to be interchangeable for QUIC_BUFFER");
```